htdocs/ticket/card.php

```diff
@@ -191,6 +191,9 @@
 
 			$object->fk_project = $projectid;
 
+			// File transfer
+			$object->copyFilesForTicket();
```

Comment:
What is the reason to move this before the database action ?
Doing it after allow to avoid to make changes on disks if there is an error on database (due to if !$error test) so nothing is done.
If you do it before, even if there is an error that need to cancel action, file system will be altered by the copyFilesForTicket.  Is there is particular reason ?

Reply:
@eldy 
I put "copyFilesForTicket" before "create" in order to have attached files on ticket card.
If you put after and attach a file, the "trackid" won't be the same and nothing will be attached on ticket card.

Reply:
> @eldy I put "copyFilesForTicket" before "create" in order to have attached files on ticket card. If you put after and attach a file, the "trackid" won't be the same and nothing will be attached on ticket card.

I experience this opposite.
copyFilesForTicket() has the line code
`$formmail->trackid = 'tic'.$this->id;`
So this->id must be defined. And to be defined, the record must be created into database.

Reply:
"trackid" is null at begin, when you add the file in the create form.
And if you put "cpyFilesForTicket" after create the "trackid" is "tic".$this->id and $this->id is set.
So the attached file is not found and not put on ticket card.
Perhaps, it will be better to cut the copyFilesForTicket() into separated methods.

Reply:
I pushed a different fix...